1) added array1 and output in console - even numbers;

matveySV opened the pull request:
1) added array1 and output in console - even numbers;
2) added enum WeekDays.java and output in console new switch mode

LGTM, please !!!!

matveySV committed:
1) added array1 and output in console - even numbers;
2) added enum WeekDays.java and output in console new switch mode

nikmikhailov13 commented on src/Main.java:
```diff
@@ -1,5 +1,24 @@
+            case WEDNESDAY -> System.out.println("Sereda");
+            case THURSDAY -> System.out.println("Chetver");
+            case FRIDAY -> System.out.println("Pyatnica");
+            case SATURDAY, SUNDAY -> System.out.println("Weekend");
```
where is the default case? :)

matveySV replied:
we use the enums and do not allow the possibility to enter something else other than enums, so we don't need default.

nikmikhailov13 reviewed (approved):
See my comments. 
Otherwise LGTM

matveySV committed:
1) added array1 and output in console - even numbers;
2) added enum WeekDays.java and output in console new switch mode
3) corrected with "Default"

matveySV committed:
1) added array1 and output in console - even numbers;
2) added enum WeekDays.java and output in console new switch mode
3) corrected with "Default"
